doc/devel/MEP/MEP14.rst

```diff
@@ -78,7 +78,7 @@ number of other projects:
 - `Microsoft DirectWrite`_
 - `Apple Core Text`_
 
-.. _pango: https://pango.gnome.org
+.. _pango: https://github.com/GNOME/pango
```

Comment:
That's a mirror; it should've gone to their GitLab.

Reply:
Or, to their website: https://www.gtk.org/docs/architecture/pango - sorry for not checking, will fix in a follow up PR